ironfish/src/rpc/adapters/httpAdapter.ts

```diff
@@ -193,21 +196,24 @@ export class RpcHttpAdapter implements IRpcAdapter {
     // so keeping that convention here. Could think of a better way to handle?
     const body = combined.length ? combined.toString('utf8') : undefined
 
+    let chunkStreamed = false
     const rpcRequest = new RpcRequest(
       body === undefined ? undefined : JSON.parse(body),
       route,
       (status: number, data?: unknown) => {
-        response.writeHead(status, {
-          'Content-Type': 'application/json',
-        })
-        response.end(JSON.stringify({ status, data }))
+        response.statusCode = status
+        const delimeter = chunkStreamed ? ',' : ''
```

Comment:
you should avoid using a delimter that can show up in the content. It makes message parsing almost impossible. You want to use something like this, which cannot show up unescaped in JSON.

https://github.com/iron-fish/ironfish/blob/master/ironfish/src/rpc/adapters/socketAdapter/protocol.ts#L6

See how it's used here https://github.com/iron-fish/ironfish/blob/44a7990c9c830a61513fdb3a0f510c3d272addba/ironfish/src/rpc/messageBuffer.ts#L31

Reply:
Yea makes sense. I originally had it the same as the TPC delimiter idk why I changed it. I'll change it back